packages/contentful--create-contentful-app/src/getTemplateSource.ts

```diff
@@ -92,6 +92,10 @@ async function makeContentfulExampleSource(options: CLIOptions): Promise<string>
     return selectTemplate(ContentfulExample.Typescript);
   }
 
+  if (options.deliveryFunction) {
```

Comment:
It is not part of the ticket description but I guess it would make sense to also implement the same logic for the `--action` flag to avoid confusion when developers try to run `create-contentful-app --action`
e.g.
```suggestion
  if (options.deliveryFunction || options.action) {
```